layouts/shortcodes/cncf-landscape.html

```diff
@@ -54,10 +54,18 @@
   /* handle initially-set fragment */
   hashChangeHandler();
 });
+
 </script>
-{{- end -}}
+{{ end }}
+{{ $category := .Get "category" }}
+{{ if eq $category "kubernetes-training-partner" }}
+<iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
+<script src="https://landscape.cncf.io/iframeResizer.js"></script>
```

Comment:
I plan to suggest a tweaked version of this. I haven't time right now - sorry.

Reply:
(what I want to do is copy the category from the parameter into the HTML)

Reply:
OK. I have not been following these changes, but you want to populate the `src` attribute with
the `category` variable? Something like:
```
<iframe src="https://landscape.cncf.io/category={{$category}}&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
```